### DISCUSSION TITLE
Love it Love Love

### DISCUSSION THREAD

**Discussion by jamesc2151** [ampache/ampache#2661] [Ideas]
Love the direction version 5 is heading....and I know you guys are busting your ass doing it...Found some little cosmetic issues not worth bother for a major problem. But wanted to help in getting it all nice and sweet..

1st one ......Program related..  

 Search box very fast but Images associated with search phrase showing a broken image even tho it truly is in the database.

2nd one.... More of a request....

On album view.  Is there any way to make the view or have the option to not include the year in breaking apart an album ,,,''IE I have say like collections of Hits of the 60's  and because the ID tags show different years per son I get an album entry for each. 

Thanks Guys..... Have a Merry Xmas and New Year..
James

**Comment by lachlan-00** [comment]
In the search box a broken image usually means your rewrite rules aren't working.

I think the links are /play/ urls. That's at least happened to me before when I've done a new copy and forgot to move the .htaccess files.

Year in columns should be a preference or at least a preference to hide columns should be added.

I can make an array of columns for artist (album list) and album (song lists) where people can hide columns

**Comment by jamesc2151** [comment]
You were correct about rewrite forgot to pull over file... LOL....
Preference to hide would be great as long as it grouped the album back
together on the change.

Thanks
M-xmas

**Reply by lachlan-00** [reply]
Yes the hiding would be visual/column based only so would not affect grouping. I think it'll make a great option
